openedx/features/philu_courseware/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from model_utils.models import TimeStampedModel
+from django.contrib.auth.models import User
+
+from opaque_keys.edx.django.models import UsageKeyField
+
+
+class CompetencyAssessmentRecord(TimeStampedModel):
+
+    COMP_ASSESSMENT_TYPES = (
+        ('pre', 'Pre Assessment'),
+        ('post', 'Post Assesment')
+    )
+
+    CORRECTNESS_CHOICES = (
+        ('correct', 'Correct'),
+        ('incorrect', 'Incorrect'),
+    )
```

Comment:
Let's move these to the `constants.py` file and import from there so they can be reused if required in the future.